data/add_coordinates.py

```
@@ -0,0 +1,210 @@
+def format_json_data(data: dict) -> str:
+    """
+    Форматує дані JSON з правильним вирівнюванням.
+
+    Args:
+        data: Словник з даними для форматування
+
+    Returns:
+        Відформатований JSON рядок
+    """
+    # Знаходимо максимальну довжину ключа для вирівнювання
+    max_key_length = max(len(key) for key in data.keys())
```

Ruff (0.12.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)